### DISCUSSION TITLE
Page View Event for Cached Pages

### DISCUSSION THREAD

**Discussion by jasonmckinney66** [digitalwithyou/craft-facebook-conversion#22] [Q&A]
I'm implementing your suggestion for firing events on cached pages using JS to retrieve the partial from a folder titled /templates/dynamic. This works beautifully when implemented from within a template where the folder can be accessed.

When implementing Page View which needs to fire on every page, I would rather not go in to every template and add the JS call. Locating the call within _layout will not work because  /templates can't be accessed from there - only /web. Any suggestions for adding this globally?

**Comment by Diewy** [comment]
Would adding it to a global Javascript file and adding that file to your template work for you?
